sstables/types.hh

```diff
@@ -498,6 +498,7 @@ enum class scylla_metadata_type : uint32_t {
     Features = 2,
     ExtensionAttributes = 3,
     RunIdentifier = 4,
+    CannotHavePartitionTombstones = 5, // true => no partition tombstones, false => we don't know
```

Comment:
this doc would be be located at the top of cannot_have_partition_tombstones()

Reply:
done